### ISSUE TITLE
Wrongly detected function boundary for Python function if comment in function contains an uneven amount of apostrophe characters

### ISSUE THREAD

**Issue by tobias-klein** [terryyin/lizard#107]
Given this code:

```
def foo():
    # What we're doing here is some stuff that causes lizard to detect function boundaries wrongly. Apparently we have to use an unequal amount of apostrophes like ''''
    return False

def bar():
    if foo == 'bar':
        return True
```

lizard does not detect the second function and calculates the following wrong results:

```
================================================
  NLOC    CCN   token  PARAM  length  location  
------------------------------------------------
       7      1     10      0       6 foo@1-7@test3.py
--------------------------------------------------------------
1 file analyzed.
```

Lizard seems to have problems with the apostrophe characters in the comment in the first function.

**Comment by terryyin** [comment]
Hi @tobias-klein , thanks! really interesting bug. looking at it...

**Comment by terryyin** [comment]
OK. It's fixed and a new release is created (1.10.5).

**Comment by tobias-klein** [comment]
Thanks for the quick feedback! I appreciate it! :+1:
